veloroutes_voies_vertes/actions.py

```diff
@@ -40,3 +41,63 @@ def split_segment(*args):
         iface.messageBar().pushInfo('Véloroutes', msg)
     except QgsProviderConnectionException as e:
         iface.messageBar().pushCritical('Véloroutes', e.args[0])
+
+
+def remove_selected_portion(*args):
+    # get arguments
+    id_iti = int(args[0])
+
+    # get project instance
+    project = QgsProject.instance()
+
+    # get layers needed
+    layerPortions = project.mapLayersByName('v_portion')
```

Comment:
```suggestion
    layer_portions = project.mapLayersByName('v_portion')
```

Désolé, j'avais cru avoir fait le commentaire hier.
Peut-on suivre la nomenclature Python concernant le nommage des variables ? Les variables sont en `snake_case`.
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names
CF les autres variables ci-dessous, tu peux avoir, je pense, une fonction "Refactor" dans ton IDE pour faire ce formatage quasi-automatique.

Reply:
d'accord